lib/dor/config.rb

```diff
@@ -75,8 +75,11 @@ def make_solr_connection(add_opts = {})
             end
           }
         },
+        :dor_services => {
+          :rest_client => Confstruct.deferred { |c| config.make_rest_client c.url, c.cert_file, c.key_file, c.key_pass }
+        },
         :sdr => {
-          :rest_client => Confstruct.deferred { |c| config.make_rest_client c.url }
+          :rest_client => Confstruct.deferred { |c| config.make_rest_client c.url, c.cert_file, c.key_file, c.key_pass }
         },
```

Comment:
The use of `rest_client` and `make_rest_client` seems to imply that `RestClient` is used, but it might help to add comments that explicitly clarify the expected client to be used.  Also, the specs might have to check that the config is using `RestClient` so that if, or when, the client library is changed out, the specs break and it will be easy to find the code locations that depend specifically on `RestClient`.